docs/sources/alerting/fundamentals/annotation-label/variables-label-annotation.md

```diff
@@ -13,15 +13,11 @@ weight: 117
 
 # How to template annotations and labels
 
-In Grafana it is possible to template annotations and labels just like you would in Prometheus. Those who have used
-Prometheus before should be familiar with the `$labels` variable which holds the label key/value pairs of the alert
-instance and the `$value` variable which holds the evaluated value of the alert instance.
+In Grafana it is possible to template annotations and labels just like you would in Prometheus. Those who have used Prometheus before should be familiar with the `$labels` variable which holds the label key/value pairs of the alert instance and the `$value` variable which holds the evaluated value of the alert instance.
```

Comment:
```suggestion
In Grafana it is possible to template annotations and labels just like you would in Prometheus.
Those who have used Prometheus before should be familiar with the `$labels` variable
which holds the label key/value pairs of the alert instance and the `$value` variable
which holds the evaluated value of the alert instance.
```

Reply:
Note that I have not changed any of the text, which needs revision because I have only changed the linebreaks.